sysl2/naive/Grammar1_test.go

```diff
@@ -219,7 +219,7 @@ func TestBuildEBNFGrammar2(t *testing.T) {
 	assert.Truef(t, result, "unable to parse text=(%s)", text)
 
 	choiceActual, s := ruleSeq(ast[0], "s")
-	assert.Truef(t, 0 == choiceActual && len(s) == 3, "parse returned incorrect ast for text=(%s)", text)
+	assert.Truef(t, choiceActual == 0 && len(s) == 3, "parse returned incorrect ast for text=(%s)", text)
```

Comment:
Leave these as is. I've reworked them considerably in another commit.

Reply:
ok, changed back